packages/playwright-core/src/utils/dependencies.ts

```diff
@@ -183,11 +201,36 @@ export async function validateDependenciesLinux(sdkLanguage: string, linuxLddDir
   }
 
   const maybeSudo = (process.getuid() !== 0) && os.platform() !== 'win32' ? 'sudo ' : '';
+  const dockerInfo = await readDockerVersion();
   const errorLines = [
     `Host system is missing dependencies to run browsers.`,
   ];
-  if (missingPackages.size && !missingDeps.size) {
-    // Happy path: known dependencies are missing for browsers.
+  // Ignore patch versions when comparing docker container version and Playwright version:
+  // we **NEVER** roll browsers in patch releases, so native dependencies do not change.
+  if (dockerInfo && !dockerInfo.driverVersion.startsWith(utils.getPlaywrightVersion(true /* majorMinorOnly */))) {
```

Comment:
```suggestion
  if (dockerInfo && !dockerInfo.driverVersion.startsWith(utils.getPlaywrightVersion(true /* majorMinorOnly */) + '.')) {
```

To avoid 1.200 vs 1.20 mismatch 😄